core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponent.scala

```diff
@@ -378,7 +383,12 @@ trait AttributeComponent {
       }
 
       def clearAttributeScratchTableAction(transactionId: String) = {
-        sqlu"""delete from #${baseTableRow.tableName}_SCRATCH where transaction_id = $transactionId"""
+        val tableSuffix = if (baseTableRow.tableName == "ENTITY_ATTRIBUTE")
+          "TEMP"
+        else
+          "SCRATCH"
+
+        sqlu"""delete from #${baseTableRow.tableName}_#${tableSuffix} where transaction_id = $transactionId"""
```

Comment:
If we're using a temp table, we shouldn't bother deleting rows; we should just drop the entire temp table, which should happen in `DataSource`. You could make this function a no-op in the temp case, or have some conditional logic in `updateAction()` just below